src/Bridge/Laravel/Console/Commands/CreateProviderCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Multitenancy\Bridge\Laravel\Console\Commands;
+
+use Illuminate\Console\Command;
+use LoyaltyCorp\Multitenancy\Bridge\Laravel\Console\Exceptions\CommandOptionUnusable;
+use LoyaltyCorp\Multitenancy\Services\Providers\Interfaces\ProviderServiceInterface;
+
+final class CreateProviderCommand extends Command
+{
+    /**
+     * Constructor for creation of a provider command.
+     */
+    public function __construct()
+    {
+        $this->description = 'Create a new provider';
+        $this->signature = 'app:provider:create 
+        {--identifier: The unique identifier for the provider to be referenced by} 
+        {--name: Human friendly name of the provider}';
```

Comment:
Is it really human friendly or is it machine friendly?

Reply:
machine friendly !== machine parsable